.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
This file allows for prettier (or another formatting tool) to ensure that everyone's changes are formatted the same to reduce errors and make PRs easier.